sklearn/utils/tests/test_sparsefuncs.py

```diff
@@ -478,3 +481,19 @@ def test_csc_row_median():
 
     # Test that it raises an Error for non-csc matrices.
     assert_raises(TypeError, csc_median_axis_0, sp.csr_matrix(X))
+
+
+def test_inplace_normalize():
+    ones = np.ones((10, 1))
+    rs = RandomState(10)
+
+    for inplace_csr_row_normalize in (inplace_csr_row_normalize_l1,
+                                      inplace_csr_row_normalize_l2):
+        for dtype in (np.float64, np.float32):
+            X = rs.rand(10, 5).astype(dtype)
```

Comment:
you should use `randn` to have negative values
That will show that you forgot the absolute value before the sum